RecoTracker/LSTCore/src/alpaka/PixelTriplet.h

```diff
@@ -1272,8 +1272,8 @@ namespace lst {
     if ((z_OutLo < zLoPointed) || (z_OutLo > zHiPointed))
       return false;
 
-    const float sdlPVoff = 0.1f / rt_OutLo;
-    sdlCut = alpha1GeV_OutLo + alpaka::math::sqrt(acc, sdlMuls2 + sdlPVoff * sdlPVoff);
+    const float lstPVoff = 0.1f / rt_OutLo;
+    lstCut = alpha1GeV_OutLo + alpaka::math::sqrt(acc, lstMuls2 + lstPVoff * lstPVoff);
```

Comment:
in this and a few other cases perhaps `lst` or `sdl` can be dropped completely and where appropriate replaced by a more clear name.
Here `dPhiCut` is more appropriate.

in the above `muls2` or below `thetaMuls2` would be clear enough

Reply:
Sure, I can change these.